java/.envrc

```diff
@@ -0,0 +1,4 @@
+# this is a publicly available password for the evergreen test jvm keystore
```

Comment:
issue: I'm not sure we should mention evergreen here as it may not have meaning to consumers of this.

Reply:
I've removed the mention of evergreen. I now just say that this is an arbitrary password.